tmail-backend/jmap/extensions/src/main/java/com/linagora/tmail/james/jmap/AttendanceStatus.java

```diff
@@ -0,0 +1,5 @@
+package com.linagora.tmail.james.jmap;
+
+public enum AttendanceStatus {
+    Accepted, Tentative, Delegated, NeedsAction, Declined
```

Comment:
```suggestion
    Accepted("$accepted"), 
    Tentative("$tentative"), 
    Delegated("$delegated"),
    NeedsAction($needs-action"), 
    Declined("$declined");
    
    private final String userFlag;
    
    public static AttendanceStatus fromUserFlag(String userFlag) {
        // ...
    }
```

And get rid of the aweful map

Reply:
Delegated is not in use now. It would additionally require some extra handling because, in addition to the delegated flag, we need to keep track of whom we are delegating our event attendance status, so I'm thinking of just removing it from the enum.